style.css

```diff
@@ -1,3 +1,428 @@
-h1 {
-  color: green;
+body {
```

Comment:
[OPTIONAL] Good job with the requested changes. 👍  But let's fix the overflow issue, kindly remove the 8px margin around the body and the page will look perfect 👍 
<img width="1010" alt="Screenshot 2023-03-25 at 2 05 44 AM" src="https://user-images.githubusercontent.com/25887253/227659422-407ba803-882d-4329-8745-0a433c5e514e.png">